integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,29 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeHostPort(t *testing.T) {
```

Comment:
Also, when you run `make test` do other tests fail? Lots of tests use the same functions so you should see a ton of failures if it really is your setup, but if they all pass then it must be something about your edits.